components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1363,6 +1328,18 @@ else if (getZCTCoords(plane)[2] < timestamps.size()) {
     segments = null;
   }
 
+  // -- ZeissCZI-specific methods --
+
+  public boolean allowAutostitching() {
+    return ((DefaultMetadataOptions) getMetadataOptions()).getBoolean(
```

Comment:
The idea of updating objects is interesting, but we'd have to talk through what we want to support. The contract that is currently in place is that a user can pass the implementation of an interface. (And that gets at least _partially_ passed around.) Updating would violate that.

What's not working in `FormatHandler` at the moment?